docs/en/manuals/live-update.md

```diff
@@ -127,218 +61,46 @@ When bundling, any excluded resource will be left out of the application bundle.
-A public/private key pair is used to cryptographically sign the manifest. Signing uses 512/1024/2048-bit RSA keys in `.der`-format that the user needs to supply. You can generate these using the `openssl` tool:
-
-```sh
-$ openssl genrsa -out private_raw.key 1024
-$ openssl pkcs8 -topk8 -inform pem -in private_raw.key -outform der -nocrypt -out private.der
-$ openssl rsa -in private_raw.key -outform DER -RSAPublicKey_out -pubout -out public.der
-```
-This will output `private_raw.key` (can be safely deleted), `private.der`, and `public.der`. To use the keys for signing open the Live update settings view and point respective fields to the generated keys.
-
-![Manifest signature key-pair](images/live-update/manifest-keys.png)
-
-### Scripting with Live update manifest
-Adding to the scripting example above, we add the following callback function
-
-```lua
-local function store_manifest_cb(self, status)
-    if status == resource.LIVEUPDATE_OK then
-        print("Successfully stored manifest!")
-    else
-        print("Failed to store manifest, status: ", status)
-    end
-end
-```
+## The .zip archives
 
-and the following to ```on_message``` to handle message ```attempt_download_manifest```:
+A liveupdate .zip file contains files that were excluded from the base game package.
+The .zip file also contains a manifest file that describes the meta data for each resource within the .zip file.
 
-```lua
-...
-elseif message_id == hash("attempt_download_manifest") then
-    local base_url = "https://my-game-bucket.s3.amazonaws.com/my-resources/" -- <1>
-    http.request(base_url .. MANIFEST_FILENAME, "GET", function(self, id, response)
-        if response.status == 200 or response.status == 304 then
-            -- We got the response ok.
-            print("verifying and storing manifest " .. MANIFEST_FILENAME)
-            resource.store_manifest(response.response, store_manifest_cb) -- <2>
-        else
-            -- ERROR! Failed to download manifest!
-            print("Failed to download manifest: " .. MANIFEST_FILENAME)
-        end
-    end)
-end
-```
-1. The manifest will be stored on S3 next to the rest of the Live update resources. As before, if you create a Zip archive with resources you need to host the files somewhere and reference their location when downloading them with `http.request()`.
-2. Similar to how resources are downloaded and stored, the call to `resource.store_manifest` takes the downloaded manifest data and a callback as arguments. The function will verify the manifest and persist it to local storage.
+While our current pipeline only supports creating a single .zip file, it is in fact possible to split that zip file into smaller .zip files. This allows for smaller downloads for a game: level packs, seasonal content etc.
```

Comment:
Could you pls add info what to do with the manifest file in case user decided to split archive into a few (I got this question from our community just a while ago)
Should the manifest be in the archive n1 or it should be unpacked and splited as well etc